service-front/module/Application/tests/Form/Lpa/TrustCorporationFormTest.php

```diff
@@ -68,11 +68,9 @@ public function testValidateByModelInvalid()
         $this->assertEquals([
             'name' => [
                 0 => 'cannot-be-blank',
-                1 => 'must-be-greater-than-or-equal:1'
```

Comment:
Changes to opg-lpa-datamodels mean that this test is now incorrect. This was not picked up previously as unit tests weren't running.